src/lib/crypto/kimchi_bindings/stubs/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "wires_15_stubs"
+version = "0.1.0"
+authors = ["opensource@o1labs.org"]
+description = "OCaml stubs for the plonk proof systems"
+repository = "https://github.com/MinaProtocol/mina"
+license = "MIT/Apache-2.0"
+edition = "2021"
+
+[lib]
+name = "wires_15_stubs"
+crate-type = ["lib", "staticlib"]
+
+[dependencies]
+array-init = "2.0.0"
+rmp-serde = "0.15.5"
+libc = { version = "0.2.0" }
+num-bigint = { version = "0.4.0" }
+paste = "1.0.5"
+rand = { version = "0.8.0" }
+rayon = { version = "1" }
+serde = "1.0.130"
+sprs = { version = "0.11.0", features = ["multi_thread"] }
+
+# arkworks
+ark-ff = { version = "0.3.0", features = ["parallel"] }
+ark-serialize = "0.3.0"
+ark-ec = { version = "0.3.0", features = ["parallel"] }
+ark-poly = { version = "0.3.0", features = ["parallel"] }
+
+# proof-systems
+commitment_dlog = { git = "https://github.com/o1-labs/proof-systems", branch = "mimoo/backend", features = ["ocaml_types"] }
```

Comment:
The `Cargo.toml` files in this PR use different branches to pull the crates from :expressionless: I'm putting the submodule back and reverting these, as we discussed OOB.

Reply:
ah cool! Yeah I was planning to fix these once you got the feature branch created. I'm cool if you want to make this one work :o but happy to do that as well